cernopendata/modules/fixtures/data/docs/cod-portal-upgrade-31-1-2024/cod-portal-upgrade-31-1-2024.md

```diff
@@ -0,0 +1,14 @@
+A new version of the CERN Open Data Repository has been deployed in January 2024.
+Some of the highlights of this release are:
+
+* Change of technologies, from Angular to [ReactJS](https://react.dev/) and from Bootstrap to [Semantic UI](https://semantic-ui.com/)
+for the web interface, which improves the navigation and overall use experience.
+* Changes in the search functionality, including modifying the default search to do an `AND` of the terms (instead of `OR`),
+escaping the special character `/` (allowing searches by DOI).
+* Redesign facets for results filtering.
+* Change the self-maintained ElasticSearch instance, which had reached its end of life support, by the open-sourced
+OpenSearch instances provided by the CERN OpenSearch service.
+* Update to the latest version of the underlying components, including the invenio framework, xrootd and docker compose.
```

Comment:
Again, open data users don't see "docker compose". The open data curators do, but these know to read the DEVELOPING guide, that we changed already. 

It is good to distinguish between (i) end-user facing news and (ii) developer-facing news. I think this item should describe the former, and we have developer guide to describe to latter.

Reply:
Since the developers will also read this, it might be worth mentioning some of the changes here. Otherwise, we could just forget about the highlights and put a single line saying `improving web interface`. 
Then, I find it difficult where to set the line. Some of the things are indeed quite internal (like python version). Others, like the fact that opendata uses the invenio framework, I find it that it is very good to remind our users. The docker seems to be the one were we have more conflicting points of view. What about we leave docker out, and all the other in?